### PR TITLE
complete task 10 + testing + error handling

### PR DESCRIPTION
**Pull request by xViolaine**
I have made an endpoint to post a comment onto a review specified by a parametric endpoint and added error handling as well as testing

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -1,289 +1,369 @@
...
-                            'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
-                        review_body: 'Fiddly fun for all the family',
-                        category: 'dexterity',
-                        created_at: new Date(1610964101251).toISOString(),
-                        votes: -15
-                    })
-                })
-        })
-    })
+  test("task 10: code 400, body does not contain both mandatory keys", () => {
```
Comment (cjplatten) [inline __tests__/app.test.js]: This test description doesn't match the test

File: __tests__/app.test.js [hunk 2]
```diff
@@ -1,289 +1,369 @@
...
+      body: "I like doggos :3",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .expect(404)
+      .send(newComment)
+      .then(({ body }) => {
+        expect(body.msg).toBe("You can't add a comment here because the review doesn't exist!");
+      });
+  });
```
Comment (cjplatten) [inline __tests__/app.test.js:355]: What response does you server give to a post request if the review id is invalid?

File: controllers/controller-games.js [hunk 3]
```diff
@@ -1,109 +1,144 @@
...
+        status: 400,
+        msg: `Body doesn't contain mandatory keys`,
+      });
+  }
 
-}
\ No newline at end of file
+  addComment(req.body, review_id)
+    .then((newComment) => {
+      res.status(201).send({ newComment });
```
Comment (cjplatten) [inline controllers/controller-games.js]: I'd recommend returning this on the key of 'comment'

**Review by cjplatten — CHANGES_REQUESTED** [review]
A few comments to address, good so far though!

File: __tests__/app.test.js (hunk 1 above)
Comment (xViolaine) [reply __tests__/app.test.js]: Thanks, changed it!

File: controllers/controller-games.js (hunk 3 above)
Comment (xViolaine) [reply controllers/controller-games.js]: Okay, changed as well!

**Pull request merged**